lib/rack/lint.rb

```diff
@@ -835,10 +835,14 @@ def close
 
       def verify_to_path
         ##
-        ## If the Body responds to +to_path+, it must return a +String+ path for the local file system whose contents are identical to that produced by calling +each+; this may be used by the server as an alternative, possibly more efficient way to transport the response. The +to_path+ method does not consume the body.
+        ## If the Body responds to +to_path+, it must return either +nil+ or a +String+.  If a +String+ is returned, it must be a path for the local file system whose contents are identical to that produced by calling +each+; this may be used by the server as an alternative, possibly more efficient way to transport the response.  The +to_path+ method does not consume the body.
```

Comment:
For consistency please do not use double space. Other than that, this PR looks great.

Reply:
@ioquatix sorry for the delay, fixed and rebased.